src/helpers/index.js

```diff
@@ -1,5 +1,21 @@
 import { format } from 'date-fns';
 
+/**
+ * Format a UTC date
+ *
+ * @param {Date} startDate UTC date
+ * @param {string} formatStr pattern to be formatted
+ * @returns {string} Formatted date
+ */
+export const formatUTCDate = (date, formatStr = 'MM/dd/yy') => {
```

Comment:
What would happen if I pass `undefined` or `null` in `date?` Are you sure this won't blow up? :thinking: